site/app/libraries/database/DatabaseQueries.php

```diff
@@ -728,25 +728,65 @@ public function toggleLikes(int $post_id, string $current_user): array {
         try {
```

Comment:
I missed it on the previous upducks PR, but this would be a great opportunity to use PHPStan's [array shape](https://phpstan.org/writing-php-code/phpdoc-types#array-shapes) feature to specify the expected form of the returned array.